### DISCUSSION TITLE
Version 0.8.4

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#778] [Announcements]
Greetings! Machinaris v0.8.4 is now available. Changes include:
- Scaling-Down: Optional mode where wallets are synced daily, not run 24/7.  Saves ~35% memory so smaller farmers can farm more blockchains concurrently on the same machine. See Wallets page, top-right Settings.
- Scaling-Up: Improved plot tracking efficiency for multi-PB sized farms. Thanks @grobalt!
- Warnings for duplicated, invalid, or key-less plots.  See Farming page.
- Update: [Flax](https://github.com/Flax-Network/flax-blockchain) to v0.1.10, [Cactus](https://github.com/Cactus-Network/cactus-blockchain) to v1.5.2, [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.6.0) to v1.6.0
- Security: Disable Setup page's mnemonic import field autocomplete from caching value in your local browser. Thanks @Baatezu!
- Fixes: Better handling of farmed block logging for certain blockchains like Apple & BPX, Alerts from Chia 1.5.1 for added coins missing due to blockchain logging changes.  Improved Smartctl response processing. Thanks @gnomuz!

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.8.4'>Version 0.8.4</a>.</em>

**Comment by Einsteinjr84** [comment]
Is there any documentation on where to set "scaling-down"?

**Reply by guydavis** [reply]
Yes, please see the [Wiki](https://github.com/guydavis/machinaris/wiki/Wallets#settings). On the Wallets page, click the Settings button (gear icon) top-right.  

![image](https://user-images.githubusercontent.com/12936/191812024-e6caab9b-fb98-4ffa-b84a-fb366b7915b9.png)

Choose 'Daily' then monitor over the next few days.  On a background thread, each blockchain wallet will be synced at a random hour thru the day, limiting the number of blockchain wallets running concurrently, which lowers the peak memory usage of multiple blockchain fullnodes on a single machine.   Hope this helps!
